src/analysis/scripts/export_connectivity.sh

```diff
@@ -75,6 +75,35 @@ function ec_export_table_csv() {
          -c "\COPY ${EXPORT_TABLENAME} TO '${FILENAME}' WITH (FORMAT CSV, HEADER)"
 }
 
+function ec_export_table_geojson() {
+  OUTPUT_DIR="${1}"
+  EXPORT_TABLENAME="${2}"
+
+  FILENAME="${OUTPUT_DIR}/${EXPORT_TABLENAME}.geojson"
+  ogr2ogr -f GeoJSON "${FILENAME}" \
+          -t_srs EPSG:4326 \
```

Comment:
Is there a reason the other geojson function includes `-s_srs "EPSG:${NB_OUTPUT_SRID}"` and this one doesn't?